tests/reporter/kubernetes.go

```diff
@@ -1208,7 +1209,7 @@ func (r *KubernetesReporter) executeVirtLauncherCommands(virtCli kubecli.Kubevir
 		{command: bridgeJVlanShow, fileNameSuffix: "brvlan"},
 		{command: bridgeFdb, fileNameSuffix: "brfdb"},
 		{command: "env", fileNameSuffix: "env"},
-		{command: "[ -f /var/run/kubevirt/passt.log ] && cat /var/run/kubevirt/passt.log", fileNameSuffix: "passt"},
+		{command: "sh", Args: []string{"-c", "cat /var/run/kubevirt/passt.log || true"}, fileNameSuffix: "passt"},
```

Comment:
Without the args thing and through a constant this looks fine to me.

A simpler alternative would be this:
```
{command: "touch /var/run/kubevirt/passt.log", fileNameSuffix: "passt"}
{command: "cat /var/run/kubevirt/passt.log", fileNameSuffix: "passt"}
```

Reply:
This is a nice hack, but it will end up creating passt log file (under artifacts directory) for non passt VMs.
Its noise we can prevent with the changes this PR suggests.

Reply:
I think the change is problematic. It is a kind of patch, used only in one case and leaves two options on how to add commands.

For example, will it make sense to change the `command` to be a slice? Will that avoid having another slice?
Another option is to create a dedicated function that treats this specific log. When it is not fitting the generic commands, we can either overload the existing or create an exception that adds something specific only for the new thing.

Reply:
There is another option: To run all commands in a shell. As we are in a test env, I think it should be fine.
If you need it for one, makes sense to be consistent and use it with all.

Reply:
Done.
Please take another look.